OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Because you're probably using VS and it insists on adding a UTF-8 BOM to files it touches.